CHANGELOG_PENDING.md

```diff
@@ -1,3 +1,5 @@
 ### Improvements
 
+- [codegen] #390: support resources with over 255 parameters
```

Comment:
Trying to remember if this was unique to resources - I think the issue applies to all generated classes not just resources.

Reply:
The mechanism being changed works only for custom resources specifically. Anything annotated with `@CustomType`.

Reply:
```
/**
 * Annotation used by a Pulumi Cloud Provider Package to mark complex types used for a Resource
 * output property.
 * <p>
 * A complex type must have a single constructor
 * marked with the @see {@link CustomType.Constructor} annotation.
 */
@Retention(RetentionPolicy.RUNTIME)
@Target(ElementType.TYPE)
public @interface CustomType {
```

Ok. Judging by the comment, the types annotated with this are not modeling Pulumi resources. They are the types used to describe outputs of resources. It sounds like the fix is doing the right thing but the CHANGELOG entry is misleading, that's no problem, I can update later on as it comes time to release.